src/client/app/components/admin/PreferencesComponent.tsx

```diff
@@ -312,6 +315,8 @@ export default function PreferencesComponent() {
 					onChange={e => makeLocalChanges('defaultHelpUrl', e.target.value)}
 				/>
 			</div>
+
+
 			<Button
```

Comment:
I think this would look better visually if there was some space above to separate from the previous item.

Reply:
Fixed in [e74a45d](https://github.com/OpenEnergyDashboard/OED/pull/1324/commits/e74a45dfd49245ab71551a31a40010a1f144186f)